.github/workflows/rust.yml

```diff
@@ -35,9 +35,6 @@ jobs:
         toolchain:
           - stable
           - nightly
-    defaults:
```

Comment:
I think https://github.com/lancedb/lance/blob/main/.github/workflows/bump-version/action.yml also needs to be updated

Reply:
fixed.